examples/persons/src/SubscribeToMore.re

```diff
@@ -0,0 +1,116 @@
+module GetAllPersons = [%graphql
+  {|
+    query getAllPersons {
+      allPersons {
+        id
+        name
+      }
+    }
+  |}
+];
+
+// NOTE: The shape of the returned data should exactly match the shape of the GetAllPersons query
+// See https://www.apollographql.com/docs/react/data/subscriptions/#subscribetomore for more information
+module NewPerson = [%graphql
+  {|
+    subscription {
+      Person {
+        node {
+          id
+          name
+        }
+      }
+    }
+  |}
+];
+
+// Defining those types and "%identity" converters below allows us to
+// write the updateQuery in pure Reason and avoid bs.raw alltogether
```

Comment:
maybe worths mentioning that we need to write these types by hand because we can't use right away those from graphql_ppx_re to not make people confused

gonna link this discussion

https://github.com/Astrocoders/reason-apollo-hooks/issues/65